src/commands/apm_cmds/publish.js

```diff
@@ -251,24 +277,35 @@ exports.task = function ({
             }
```

Comment:
We can edit the error messages above pointing to using the new `--version-bump` flag

Reply:
The thing with this error message is that it's getting pretty long and is already actually not readable when it's being rendered by the CLI (as it gets cut off). 

I'll see what I can do.